<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabhklsolver</class>
 <widget class="QWidget" name="tabhklsolver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="vL_solve_1">
     <item>
      <widget class="QLabel" name="l_solve_hkl">
       <property name="text">
        <string>HKL文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="lV_solve_hkl">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hL_solve_pB1">
       <item>
        <widget class="QPushButton" name="pB_solve_choose_hkl">
         <property name="text">
          <string>选择HKL文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_solve_hkl_count">
            <property name="text">
                <string>已选0个</string>
            </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vL_solve_2">
     <property name="topMargin">
      <number>25</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="hL_solve_ins">
       <item>
        <widget class="QPushButton" name="pB_solve_ins">
         <property name="text">
          <string>选择INS文件</string>
         </property>
        </widget>
       </item>
          <item alignment="Qt::AlignVCenter">
              <widget class="QLabel" name="l_solve_ins">
                  <property name="text">
                      <string>未选择INS文件</string>
                  </property>
              </widget>
          </item>
      </layout>
     </item>
        <item>
            <widget class="QLineEdit" name="lE_solve_xtparam">
                <property name="placeholderText">
                    <string>此处输入shelxt运行时参数</string>
                </property>
            </widget>
        </item>
        <item>
            <spacer name="verticalSpacer_2">
                <property name="orientation">
                    <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                    <size>
                        <width>20</width>
                        <height>40</height>
                    </size>
                </property>
            </spacer>
        </item>
     <item>
      <layout class="QHBoxLayout" name="hL_solve">
       <item>
        <widget class="QPushButton" name="pB_solve">
         <property name="text">
          <string>开始求解</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="QLabel" name="l_solve">
         <property name="text">
          <string>未开始求解</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="bar_solve">
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vL_solve_3">
     <item>
      <widget class="QLabel" name="l_solve_res">
       <property name="text">
        <string>RES文件</string>
       </property>
      </widget>
     </item>
        <item>
            <widget class="QListView" name="lV_solve_res">
                <property name="selectionMode">
                    <enum>QAbstractItemView::ExtendedSelection</enum>
                </property>
            </widget>
        </item>
     <item>
      <layout class="QHBoxLayout" name="hL_solve_pB2">
       <item>
        <widget class="QPushButton" name="pB_solve_send">
         <property name="text">
          <string>发送到匹配器</string>
         </property>
        </widget>
       </item>
       <item>
           <widget class="QLabel" name="l_solve_res_count">
               <property name="text">
                   <string>总计0个</string>
               </property>
           </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
